x-pack/docs/en/security/securing-communications/update-tls-certificates.asciidoc

```diff
@@ -0,0 +1,574 @@
+==== Generate a new certificate for each node in your cluster
+Now that your CA truststore is updated, use your new CA certificate to sign
+a certificate for each node in your cluster. Rather than run the
+`elasticsearch-certutil` tool once for each node, create an `instances.yml` file
+that contains information about each of your nodes. You can then use the
+`elasticsearch-certutil` tool to generate certificates for all of your nodes
+with a single command.
+
+. On any node in your cluster, create an `instances.yml` file that contains
+information about each of your nodes. Include all IP addresses and DNS names
+that you use to identify each node. 
++
+[source,yaml]
+----
+instances:
+  - name: "node1"
+    ip:
+      - "192.168.1.101"
+    dns:
+      - "node1"
+      - "node1.mydomain.com"
+  - name: "node2"
+    ip:
+      - "192.168.1.102"
+    dns:
+      - "node2"
+      - "node2.mydomain.com"
+  - name: "node3"
+    ip:
+      - "192.168.1.103"
+    dns:
+      - "node3"
+      - "node3.mydomain.com"
+  - name: "node4"
+    ip:
+      - "192.168.1.104"
+    dns:
+      - "node4"
+      - "node4.mydomain.com"
+----
+
+. Using the new CA certificate and key, create a new certificate for each node
+in your cluster. Include the `instances.yml` file that you created previously.
++
+[source,shell]
+----
+./bin/elasticsearch-certutil cert --in instances.yml --ca-cert ca/ca.crt \
+--ca-key ca/ca.key
+----
++
+[%collapsible%open]
+.Command parameters
+====
+   `-in`:: Specifies the name of the input YAML file (`instances.yml`)
+   containing information about each of your nodes.
+
+   `--ca-cert`:: Specifies the path to your new CA certificate (`ca.crt`) in PEM
+   format. You must also specify the `--ca-key` parameter.
+
+   `--ca-key`:: Specifies the path to the private key (`ca.key`) for your CA
+   certificate. You must also specify the `--ca-cert` parameter.
+====
+
+   a. Enter a name for the output file or accept the default of
+   `certificate-bundle.zip`.
+
+   b. When prompted, enter a password for each node certificate. It's good
+   practice to use a different password for each node certificate and store the
+   files in a secure location.
+
+   c. Decompress the `certificate-bundle.zip` file. The output contains a
+   directory for each of your nodes with a `<node-name>.p12` file.
+
+   d. Rename the `<node-name>.p12` file to match the existing file name
+   specified by the `xpack.security.transport.ssl.keystore.path` variable in 
+   the `elasticsearch.yml` file.
+
+. On *every* node, copy the appropriate node certificate to the `ES_PATH_CONF`
+directory.
++
+NOTE: The `ES_PATH_CONF` variable is the path for the {es}
+configuration files. If you installed {es} using archive distributions
+(`zip` or `tar.gz`), the variable defaults to `ES_HOME/config`. If you used
+package distributions (Debian or RPM), the variable defaults to `/etc/elasticsearch`.
+
+. Delete the old node certificate from each node. 
```

Comment:
```suggestion
. Delete the old node keystore from each node. 
```

Reply:
We instructed users to overwrite the keystore so there's prbably nothing to delete right?